packages/sdk/src/entities/Chain.ts

```diff
@@ -27,6 +32,10 @@ export class Chain {
 
   connection?: ChainConnectionDetails;
 
+  label: string; // Convenience property
+  nativeTokenName: string; // Convenience property
```

Comment:
no longer need if change on L42 in "packages/frontend/helpers/chains.ts" is accepted